community.html

```diff
@@ -80,13 +80,40 @@ <h3>More Solid Events</h3>
                     <div class="col-md-12">
                         <div class="accordion" id="accordionExample">
                             <div class="accordion-item">
-                                <h2 class="accordion-header" id="headingOne">
+                                <h2 class="accordion-header" id="headingTwentyTwentyFour">
```

Comment:
I strongly advise using numerals for year-based things, instead of words, such as
```suggestion
                                <h2 class="accordion-header" id="heading2024">
```
I would only resort to words if numerals are not permitted by the technology in question. In such case, I would strongly advise using more formal wording, such as `headingTwoThousandTwentyFour`, to avoid any confusion such as thinking that `TwentyTwentyFour` was accidental and should have been `TwentyFour`.